config/prod/kristen-bucket.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
```

Comment:
@kkdang  The bucket name will be automatically generated by AWS however it will contain this string in the name.  It'll look something like this..  `kristen-bucket-s3-synapseencryptedexternalbucket-t8ztderc4m13`.  Please let us know if you are ok with this or you prefer to have a specific bucket name.

Reply:
I'm not certain how to respond. How will the bucket name be exposed to potential users, if at all?

Reply:
the bucket name will be emailed to you after it gets provisioned.  then you need to give that info to the user that wants to access the bucket.